### PR TITLE
Fix code formatting in ca1008.md

### PR DESCRIPTION
**Pull request by 0liver**
## Summary

The code formatting in the second table row under https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1008#additional-zero-value-field-names was broken, so I fixed it.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/fundamentals/code-analysis/quality-rules/ca1008.md](https://github.com/dotnet/docs/blob/317033f8bd8e914681d30dd534abe4e7a24d3115/docs/fundamentals/code-analysis/quality-rules/ca1008.md) | [CA1008: Enums should have zero value](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1008?branch=pr-en-us-39221) |

### CODE REVIEW

**Comment by gewarren** [comment]
Thanks @0liver but this is intentional. If it's code-formatted, the vertical pipe character doesn't display as such.

image

**Comment by 0liver** [comment]
How about escaping the pipe then, e.g. using `&#x7c;` as mentioned [here](https://stackoverflow.com/a/23724163/177710)?

**Comment by gewarren** [comment]
Okay, maybe this is a bug in our Markdown flavor then, because I see this when I escape the pipe:

image

I'll log a bug.

File: docs/fundamentals/code-analysis/quality-rules/ca1008.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ In .NET 7 and later versions, you can configure other allowable names for a zero
 | Option value | Summary |
 | --- | --- |
 | `dotnet_code_quality.CA1008.additional_enum_none_names = Never` | Allows both `None` and `Never` |
-| 'dotnet_code_quality.CA1008.additional_enum_none_names = Never&#124;Nothing' | Allows `None`, `Never`, and `Nothing` |
+| `dotnet_code_quality.CA1008.additional_enum_none_names = Never\|Nothing` | Allows `None`, `Never`, and `Nothing` |
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/quality-rules/ca1008.md:84]:
```suggestion
| `dotnet_code_quality.CA1008.additional_enum_none_names = Never|Nothing` | Allows `None`, `Never`, and `Nothing` |
```

**Review by gewarren — APPROVED** [review]

**Comment by 0liver** [comment]
Thanks for looking into this deeper! I appreciate it.

**Pull request merged**